index.d.ts

```diff
@@ -1,45 +1,76 @@
 /// <reference types="node" />
-import { IncomingMessage, ServerResponse, Server } from 'http';
-import { FastifyRequest, FastifyPluginCallback, RawServerBase, RawServerDefault, RawRequestDefaultExpression, RawReplyDefaultExpression, RequestGenericInterface, ContextConfigDefault, FastifyInstance } from 'fastify';
-import * as fastify from 'fastify';
-import * as WebSocket from 'ws';
-import { Duplex } from 'stream';
-import { FastifyReply } from 'fastify/types/reply';
+import { IncomingMessage, ServerResponse, Server } from "http";
+import {
+  FastifyRequest,
+  FastifyPluginCallback,
+  RawServerBase,
+  RawServerDefault,
+  RawRequestDefaultExpression,
+  RawReplyDefaultExpression,
+  RequestGenericInterface,
+  ContextConfigDefault,
+  FastifyInstance,
+} from "fastify";
+import * as fastify from "fastify";
+import * as WebSocket from "ws";
+import { Duplex } from "stream";
+import { FastifyReply } from "fastify/types/reply";
+import { RouteGenericInterface } from "fastify/types/route";
 
-interface WebsocketRouteOptions {
-  wsHandler?: WebsocketHandler
+interface WebsocketRouteOptions<
+  RawServer extends RawServerBase = RawServerDefault,
+  RawRequest extends RawRequestDefaultExpression<RawServer> = RawRequestDefaultExpression<RawServer>,
+  RequestGeneric extends RequestGenericInterface = RequestGenericInterface
+> {
+  wsHandler?: WebsocketHandler<RawServer, RawRequest, RequestGeneric>;
 }
-declare module 'fastify' {
+declare module "fastify" {
```

Comment:
I swear to god Prettier is gonna kill me one day

Reply:
> 
> 
> I swear to god Prettier is gonna kill me one day

If you can revert these style changes then i'll approve a workflow run, or else `standard` is going to throw a fit during the CI tests.

Reply:
Okay